ndcube/mixins/plotting.py

```diff
@@ -157,7 +185,23 @@ def _animate_cube(self, wcs, plot_axes=None, axes_coordinates=None,
             for axis_unit, coord_name in zip(axes_units, wcs.world_axis_physical_types):
                 coord_params[coord_name] = {'format_unit': axis_unit}
 
+        # TODO: `_plot_2D_cube` transposes the array if 'y' is before 'x'.
+        # Investigate if this should be the case here, or if it shouldn't be done at all.
```

Comment:
Is this done?

Reply:
it is not done, it's still an open question. I am not sure it's easy to do in the animator, and it feels weird to have it supported in 2D but not the animator?

Reply:
I think the more desired behavior is that the data is transposed when `'y'` is before `'x'`.  So the fact that it is difficult/not currently possible with the animator isn't a reason for it not to be done here.  I'd consider that a bug in animator rather than a design choice.  That's my feeling, anyway.

Reply:
In that case we should probably raise a warning here until we fix it or something.

Reply:
Sounds good.

Reply:
A warning still needs to be raised here I think?